book/IMaEA.md

```diff
@@ -4199,11 +4199,11 @@ The reason for using memory mapped files was for the convenience of converting o
 
 ### Writing Files
 
-Exiv2 is very reliable at writing files which conform to standards.  The way in which this is achieved is to by calling image->writeMetata() which delegates to the handlers writeMetata().
+Exiv2 is very reliable at writing files which conform to standards.  The way in which this is achieved is to by calling image->writeMetadata() which delegates to the handlers writeMetadata().
 
-Because the handler understands the structure of the image, he writes a temporary in memory copy of the image.  It proceeds to parse the image and copy the data to the temporary file.  When it arrives at each of the four metadata blocks (Exif, ICC, IPTC and XMP) it calls the serializer to create a buffer of data which is injected into the temporary image.  When it arrives the EOF on the original file, if no error has been detected it calls io->transfer() on the temporary image.  The operation transfer() copies the bytes from the temporary stream to the permanent file.
+Because the handler understands the structure of the image, it writes a temporary in memory copy of the image.  It proceeds to parse the image and copy the data to the temporary file.  When it arrives at each of the four metadata blocks (Exif, ICC, IPTC and XMP) it calls the serializer to create a buffer of data which is injected into the temporary image.  When it arrives the EOF on the original file, if no error has been detected it calls io->transfer() on the temporary image.  The operation transfer() copies the bytes from the temporary stream to the permanent file.
```

Comment:
The spell checker is tuned to only look at 3+ character words, and this was a word, but I was "in the neighborhood". I think this is the only grammatical fix I've made. In general, I try not to mix grammar fixes into spelling PRs.

I don't currently have any plans to do a round of grammar fixes (spelling fixes like this PR are relatively cheap between the tooling selecting candidates and Sheets suggesting replacements)...